src/bufr/BufrParser/Exports/Variables/TimeoffsetVariable.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * (C) Copyright 2022 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <memory>
+
+#include "eckit/config/LocalConfiguration.h"
+
+#include "Variable.h"
+
+
+namespace Ingester
+{
+    /// \brief Exports parsed data as datetimes using speciefied Mnemonics
+    class TimeoffsetVariable final : public Variable
+    {
+     public:
+        TimeoffsetVariable() = delete;
+        TimeoffsetVariable(const std::string& exportName,
+                         const std::string& groupByField,
+                         const eckit::LocalConfiguration& conf);
+        ~TimeoffsetVariable() final = default;
+
+        /// \brief Get the configured mnemonics and turn them into datetime strings
+        /// \param map BufrDataMap that contains the parsed data for each mnemonic
+        std::shared_ptr<DataObjectBase> exportData(const BufrDataMap& map) final;
+
+        /// \brief Get a list of queries for this variable
+        QueryList makeQueryList() const final;
+
+     private:
+        // \\brief This is something that will be removed later.
```

Comment:
Comment is innacurate.

Reply:
Ron, I put this here because I thought you told Praveen that this item would be deprecated. If you would please suggest an exact wording change next time, it would be beneficial.